lib/typeprof/code_range.rb

```diff
@@ -55,11 +55,11 @@ def initialize(first, last)
       raise unless first
     end
 
-    def self.from_node(node)
+    def self.from_node(node, encoding = Encoding::UTF_16LE)
```

Comment:
Is it better to make the encoding configurable during server initialization?
I don’t have a good idea on where to store this option.

Reply:
I think the reason of UTF-16 is because LSP uses it by default, right? I don't think there is any particular need to be able to make it configurable.